### PR TITLE
Create nft-rfc-008.md

### PR DESCRIPTION
**Pull request by eric-schuh**
Adding first, working draft of nft-rfc-008.

### CODE REVIEW

**Comment by jandrieu** [comment]
For consistency with the layer model, the title should probably be NFT-RFC-008 NFT Assertions

File: nft-rfc-008.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+nft-rfc: 8
+title: NFT Resource Verification
```
Comment (jandrieu) [inline nft-rfc-008.md:3]: title: NFT Assertions

File: nft-rfc-008.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+stage: draft
+category: NFT/METADATA
+authors: Shaun Conway @ig-shaun, Eric Schuh @eric-schuh
+created: 2021-1-20
+modified: 
+---
+
+# NFT RFC 008 NFT Resource Verification
```
Comment (jandrieu) [inline nft-rfc-008.md]: # NFT RFC 008 NFT Assertions

File: nft-rfc-008.md (hunk 1 above)
Comment (ig-shaun) [inline nft-rfc-008.md:3]: Change title to NFT Assertions

**Review by ig-shaun — CHANGES_REQUESTED** [review]
Needs to be updated with the redraft from Docs.

**Review by ig-shaun — APPROVED** [review]
Good first draft

**Pull request merged**